docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,501 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b08144d7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "labels[example_idx]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f62da67",
+   "metadata": {},
+   "source": [
+    "`predictions` is a list of numpy arrays where `predictions[i]` is of shape `(M,)` where `M` is the number of classes and `predictions[i][m]` is of shape `(L,5)` where `L` is the number of bounding boxes for class `k` and the five columns correspond to ``[x1,y1,x2,y2,pred_prob]`` returned by the model. Here `pred_prob` is the model confidence in the predicted label. Our example uses `k == 5` classes which is why `predictions[0].shape = (5,)`.\n",
+    "\n",
+    "An example output of `predictions[i]` is below:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3d70bec6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "predictions[example_idx]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cf95ea28",
+   "metadata": {},
+   "source": [
+    "\n",
+    "Once you have pred_probs and labels in the appropriate formats, you can **find label issues with cleanlab for any object detection dataset**!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3daff923",
+   "metadata": {},
+   "source": [
+    "## 3. Use cleanlab to find label issues\n",
+    "Based on the given `labels` and model `predictions` from our trained model, cleanlab can quickly help us find examples that contain label errors in the dataset. In object detection, label errors are annotations that are more likely imperfect.\n",
+    "\n",
+    "Examples are imperfect when annotators:\n",
+    "\n",
+    "- overlooked an object (missing annotated bounding box),\n",
+    "- chose the wrong class label for an annotated box in the correct location,\n",
+    "- imperfectly annotated the location/edges of a bounding box.\n",
+    "\n",
+    "\n",
+    "For cleanlab, any of these annotation errors should lead to an image with a lower label quality score."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3cceeefb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "num_examples_to_show = 5"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4caa635d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "label_issue_idx = find_label_issues(labels, predictions, return_indices_ranked_by_score=True)\n",
+    "label_issue_idx[:num_examples_to_show]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "638a9dc4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "is_label_issue = find_label_issues(labels, predictions)\n",
+    "is_label_issue[:num_examples_to_show]"
+   ]
+  },
```

Comment:
```suggestion
```

Reply:
never include two different ways of accomplishing the same thing like this without explanation, it can be quite confusing